_posts/2017-07-24-minority-report.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: >
+  Minority report
+author: riikkanen
+excerpt: The pictures used in recruitment advertisements show always a partial glimpse of everyday life in Solita. A little more in this minority report.
```

Comment:
Pitäisikö sanajärjestystä hieman muokata? "The pictures used in recruitment advertisements show always a partial glimpse of everyday life in Solita."
=> "The pictures used in recruitment advertisements always show a partial glimpse of everyday life in Solita."